src/librustc_mir/interpret/eval_context.rs

```diff
@@ -383,40 +383,29 @@ impl<'a, 'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> EvalContext<'a, 'mir, 'tcx, M
     ) -> EvalResult<'tcx> {
         ::log_settings::settings().indentation += 1;
 
-        /// Return the set of locals that have a storage annotation anywhere
-        fn collect_storage_annotations<'mir, 'tcx>(mir: &'mir mir::Mir<'tcx>) -> HashSet<mir::Local> {
-            use rustc::mir::StatementKind::*;
-
-            let mut set = HashSet::new();
```

Comment:
Would be a good idea to check for `Hash{Map,Set}` used by miri code: if it needs to be a hash map/set, it should be `FxHash{Map,Set}` instead.

Reply:
Done, found a few in `interpret::memory`